kong.conf.default

```diff
@@ -1989,3 +1989,105 @@
                         # * No recursion is performed. Only .wasm files at the
                         #   top level are registered
                         # * This path _may_ be a symlink to a directory.
+
+#------------------------------------------------------------------------------
+# WASM injected directives
+#------------------------------------------------------------------------------
+
+# The Nginx Wasm module has its own set of directives which can be tuned via
+# special `nginx_wasm_*` directives.
+#
+# The following namespaces are supported:
+#
+# - `nginx_wasm_<directive>`: Injects `<directive>` into the `wasm {}` block.
+# - `nginx_wasm_shm_<name>`: Injects `shm_kv <name>` into the `wasm {}` block,
+#    allowing operators to define custom shared memory zones which are usable by
+#    the `get_shared_data`/`set_shared_data` Proxy-Wasm SDK functions.
+# - `nginx_wasm_wasmtime_<flag>`: Injects `flag <flag>` into the `wasmtime {}`
+#    block, allowing various Wasmtime-specific flags to be set.
```

Comment:
Unless you think it will be more common than not for a user to set specific flags for the Wasm runtime, I think I'd rather leave it out (for brevity). It's discoverable in the [ngx_wasm_module docs](https://github.com/Kong/ngx_wasm_module/blob/main/docs/DIRECTIVES.md#flag), and my gut feeling is that if a user already knows they need to set a Wasmtime flag, they'll already have traversed these docs (or know how to search for them).